app/src/main/res/layout/fragment_comment_dialog.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="bottom"
+    android:layout_marginBottom="?attr/actionBarSize">
+
+    <EditText
+        android:id="@+id/etComment"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/add_a_comment"
+        android:textSize="14sp"
+        android:layout_marginEnd="80dp"/>
+
+    <Button
+        android:id="@+id/btnComment"
+        android:layout_width="wrap_content"
+        android:layout_height="40dp"
+        android:text="@string/post"
+        android:layout_alignTop="@+id/etComment"
+        android:layout_toEndOf="@id/etComment"
+        android:layout_marginStart="-80dp"
+        android:textSize="14sp"
+        android:textColor="@android:color/white"
+        android:backgroundTint="#458eff"
```

Comment:
again, you're using this color a lot! It'd help if you give it a readable name & save it for future usages